conan/api/subapi/cache.py

```diff
@@ -214,7 +214,7 @@ def restore(self, path):
                 unzipped_metadata_folder = pref_bundle.get("metadata_folder")
                 if unzipped_metadata_folder:
                     # FIXME: Restore metadata is not incremental, but destructive
-                    out.info(f"Restore: {pref} metadata in {unzipped_metadata_folder}")
+                    out.info(f"Restore: {pref} metadata from {unzipped_metadata_folder}")
```

Comment:
It is not restoring from, it is restoring in, the folder is the destination folder